### PR TITLE
Addressed teams feedback in multiple sections of Getting Started Readme

### PR DESCRIPTION
**Pull request by sunithasomanna**
- Pluginpath should not be empty. Ada a note in odim-controller config parameters table!
- Re-arrange steps in "Setting up the environment" section.
- Replace controller nodes with cluster nodes in "Deployment considerations" section.
- Update parameters list for all plugins in step 12 example. (including ACI plugin)
- Specify what is odim_username and odim_password in the section "Adding a plugin into the Resource Aggregator for ODIM framework"
- Remove $ symbols in commands found in lenovo deployment instructions section.
- Update few pkgs version information given along with the example of calico name and version
- Remove hardcoded connectionMethodID in sample payload of all plugins in the section "Adding a plugin into the Resource Aggregator for ODIM framework".
- To verify the added plugins, need to call 2 APIs. Rephrase to this - " A link of the executed task. Performing a GET operation on this task displays the task monitor associated with this operation in the response header".
- Add a note for MessageBusType saying RedisStreams is supported by ODIM and not plugins.
- Minor grammatical error fixes.

### CODE REVIEW

**Review by AngithCJ — APPROVED** [review]

File: README.md [hunk 1]
```diff
@@ -1577,12 +1597,19 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
-      odimPluginPath: /home/bruce/plugins
+       odimPluginPath: /home/bruce/plugins
+       odimra:
+         groupID: 2021
+         userID: 2021
+         namespace: odim
+         fqdn:
+         rootServiceUUID:
+         haDeploymentEnabled: True
        connectionMethodConf:
```
Comment (sukhwinder20) [inline README.md]: connection method should be under odimra

File: README.md [hunk 2]
```diff
@@ -1577,12 +1597,19 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
        - ConnectionMethodType: Redfish
          ConnectionMethodVariant: Compute:BasicAuth:LENOVO_v1.0.0
-         odimraKafkaClientCertFQDNSan: lenovoplugin, lenovoplugin-events
-         odimraServerCertFQDNSan: lenovoplugin, lenovoplugin-events
+       odimraKafkaClientCertFQDNSan: lenovoplugin, lenovoplugin-events
```
Comment (sukhwinder20) [inline README.md]: odimraKafkaClientCertFQDNSan and odimraServerCertFQDNSan should be under odimra

File: README.md [hunk 3]
```diff
@@ -1560,11 +1578,13 @@ Kubernetes cluster is set up and the resource aggregator is successfully deploye
...
+      ```
+      cd ~/ODIM/odim-controller/scripts
+      ```
 
 11. Open the `kube_deploy_nodes.yaml` file.
 
-         $ vi kube_deploy_nodes.yaml
+        vi kube_deploy_nodes.yaml
 
 12. Update the following parameters in the `kube_deploy_nodes.yaml` file to their corresponding values:
```
Comment (sukhwinder20) [inline README.md]: change update to append

File: README.md (hunk 1 above)
Comment (sunithasomanna) [reply README.md]: done

File: README.md (hunk 2 above)
Comment (sunithasomanna) [reply README.md]: done

File: README.md (hunk 3 above)
Comment (sunithasomanna) [reply README.md]: done

**Review by sukhwinder20 — APPROVED** [review]
LGTM

**Review by amar-shalgar — APPROVED** [review]

**Review by jeevan-kamkar — APPROVED** [review]

**Review by sukhwinder20 — APPROVED** [review]
LGTM

**Review by shivacharanms — APPROVED** [review]

**Pull request merged**